apps/docs/components/react/getting-started.md

```diff
@@ -117,8 +117,30 @@ Finally, you'll need to add CSS directives to add each Tailwind layer to `src/ap
 
 </SourceCode>
 
-::: tip
-In Next.js 13 environments that are not using [App Router](https://beta.nextjs.org/docs/api-reference/next-config#appdir) there is an issue with [Next.js not detecting ESM modules of subdependencies correctly.](https://github.com/vercel/next.js/issues/39375)
+### Next.js 13 Notes
+
+#### With App Directory
+
+Next.js 13 introduces a new [`app/`](https://beta.nextjs.org/docs/api-reference/next-config#appdir) directory. When using the App Directory, your project will default to using Server Components. However, components from Storefront UI need to be used in [client components](https://beta.nextjs.org/docs/rendering/server-and-client-components#client-components).
+
+This means that you'll have to add the [`"use client"`](https://beta.nextjs.org/docs/rendering/server-and-client-components#convention) directive at the top of any file that imports Storefront UI components.
```

Comment:
Shouldn't we remove `--experimental-app` flag [from line 28 of this file](https://github.com/vuestorefront/storefront-ui/pull/2558/files#diff-91f9521137b5f0101cbd2aa9fa2e670f3696a23e7c9e0caf2a6f5863e18eb5e1R28)?

Reply:
should we? I honestly don't know what React devs prefer. @skirianov any thoughts?